packages/dashboard/src/components/sidePanel/index.tsx

```diff
@@ -5,8 +5,14 @@ import ThresholdsSection from './sections/thresholdsSection';
 import ChartSettings from './sections/chartSettings';
 import DataSettings from './sections/dataSettings';
 import './index.css';
+import { useSelector } from 'react-redux';
+import { DashboardState } from '../../store/state';
 
 const SidePanel = () => {
+  const selectedWidgets = useSelector((state: DashboardState) => state.selectedWidgets);
+  if (selectedWidgets.length !== 1) {
+    return <div>---</div>;
+  }
```

Comment:
It's not obvious what the purpose of this code is. Could you make it clear you can only change settings on 1 widget?

Reply:
Updated.